internal/resources/credential/credential.go

```diff
@@ -73,6 +73,7 @@ var dataSpec = &schema.Schema{
 	Optional:    true,
 	MaxItems:    1,
 	Description: "Holds credentials sensitive data",
+	Sensitive:   true,
```

Comment:
Should the docs be generated again for this addition in schema?

Reply:
Yes, docs should be generated, the generation of docs can be done when next doc update is scheduled.